.circleci/config.yml

```diff
@@ -12,6 +12,18 @@ shared:
       key: ccache-{{ arch }}-{{ .Environment.CIRCLE_JOB }}-{{ .Branch }}-{{ .BuildNum }}
       paths:
         - ~/.ccache
+  restore_homebrew_cache: &restore_homebrew_cache
+    restore_cache:
+      keys:
+        - homebrew-{{ arch }}-{{ .Environment.CIRCLE_JOB }}-{{ .Branch }}
+        - homebrew-{{ arch }}-{{ .Environment.CIRCLE_JOB }}
+        - homebrew-{{ arch }}
+  save_homebrew_cache: &save_homebrew_cache
+    save_cache:
+      key: homebrew-{{ arch }}-{{ .Environment.CIRCLE_JOB }}-{{ .Branch }}-{{ .BuildNum }}
+      paths:
+        - ~/Library/Caches/Homebrew
+        - /usr/local/Homebrew
```

Comment:
Caching `~/Library/Caches/Homebrew` sounds good, but does `/usr/local/Homebrew` ever exist?

Reply:
This is where homebrew actually downloads its metadata data to. That
reduces setting up by about 5m.